week2-section2/exercise2/Spotify.cpp

```diff
@@ -0,0 +1,181 @@
+#include "Spotify.h"
+#include "PlaylistContainer.h"
+#include <string>
+
+FSpotify::FSpotify()
+{
+	ActiveMenuCommand = EMenuCommand::MainMenu;
+	PlaylistContainer = FPlaylistContainer(0);
+};
+
+//------------------------------------------------------
+void FSpotify::OpenMainMenu() 
+{
+	std::system("cls");
+	std::cout << "Welcome to Spotify! What would you like to do?\n1 - Create a Playlist\n2 - Browse Playlists\n3 - Exit" << std::endl;
+	int InputConverter;
+	std::cin >> InputConverter;
+	while (std::cin.fail() || (InputConverter <= 0) || (InputConverter >= 4))
+	{
+			std::system("cls");
+			std::cin.clear();
+			std::cin.ignore(100, '\n');
+			std::cout << "Invalid Option, Insert a valid option" << std::endl;
+			std::cout << "Welcome to Spotify! What would you like to do?\n1 - Create a Playlist\n2 - Browse Playlists\n3 - Exit" << std::endl;
+			std::cin >> InputConverter;
+	}
+
+	ActiveMenuCommand = (EMenuCommand)InputConverter;
+	std::system("cls");
+	FSpotify::RunApp();
+};
+
+//--------------------------------------------------------
+void  FSpotify::OpenCreatePlaylistMenu() 
+{
+	//Creating the playlist
+	std::system("cls");
+	std::cout << "Please enter a name for the playlist: " << std::endl;
+	char TitlePlaylist[500];
+	std::cin.getline(TitlePlaylist, 500,'\n');
+	std::cin >> TitlePlaylist;
```

Comment:
You're getting the whole line and assigning it to the TitlePlaylist char array and then you're taking the input again and assigning it again to the TitlePlaylist. 

There is no need to call std::cin >> TitlePlaylist if you've already done a getline.

Reply:
You didn't address this either 👀